src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -88,6 +89,19 @@ public class PropertiesMacro extends AbstractMacro {
     /** The file of the current module being processed. */
     private static File currentModuleFile = new File("");
 
+    /** Set of properties not inherited from the base token configuration. */
+    private static final Set<String> NON_BASE_TOKEN_PROP = Collections.unmodifiableSet(
+            Arrays.stream(new String[] {
+                "AtclauseOrder - target",
+                "DescendantToken - limitedTokens",
+                "IllegalType - memberModifiers",
+                "MagicNumber - constantWaiverParentToken",
+                "MultipleStringLiterals - ignoreOccurrenceContext",
```

Comment:
All other modules are also can be converted with such update?

Reply:
I just investigated, and this is the only one pending for macros property migration among the list. So, the presence of other checks is redundant. However, my idea was to maintain consistency, inline with non-baset-token-properties. it wasn't calling the methods that would write links of the token. So, the alternative could be just have MagicNumberCheck as a part of the constant.

Reply:
Update: The current update would help for AtclauseOrder and MagicNumber.

Reply:
Please remove them from this list is this does not help them

Reply:
Please migrate AtclauseOrder in this PR

Reply:
done